tox.ini

```
@@ -15,3 +15,9 @@ commands = python -m unittest discover tests/integration
 [testenv:example]
 passenv = POWERWALL_IP,POWERWALL_PASSWORD
 commands = python examples/example.py
+
+[testenv:server]
+changedir = .server
+commands =
+    pip install pyOpenSSL aiohttp_session aiofiles
```

There's gotta be a better way to do this...

---

```suggestion
deps = pyOpenSSL aiohttp_session aiofiles
```
I think that should work